server.js

```diff
@@ -1,14 +1,17 @@
+'use strict';
+var http = require( 'http' );
+var trucks = require('./trucks');
+var dateObj = require('./enhanced-date');
+var truckList = trucks.filterTrucksByDay(dateObj.getDayName());
```

Comment:
Because a server can run for days, you want to do this inside the request handler for the server. By doing this out here, the request handler will always return a list for the day when the server was started.